uosc.lua

```diff
@@ -1052,8 +1056,12 @@ function Menu:open(items, open_item, opts)
 			local estimated_max_width = 0
 			for _, item in ipairs(this.items) do
 				local spacings_in_item = item.hint and 3 or 2
+				local has_submenu = item.items ~= nil
+				-- M as a stand in for icon
+				local rhs = item.hint or (has_submenu and 'M' or nil)
+				local rhs_size = item.hint and this.font_size_hint or this.font_size
```

Comment:
Use words. I can't tell what `rhs` stands for 😶.

Reply:
rhs stands for right hand side. I changed it to hint_icon.